sparkling-water-droplet/build.gradle

```diff
@@ -178,3 +178,50 @@ shadowJar {
         include(dependency("org.eclipse.jetty:jetty-plus:8.1.17.v20150415"))
     }
 }
+
+// Builds a fat jar with the whole application plus a launcher as main class
+// For now the main Spark app has to be named water.SparklingWaterApp
+task externalLauncher(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar) {
+    group = "shadow"
+    description = "Builds a SW app with a launcher which can be used by external services."
+    manifest.attributes 'Main-Class': 'water.messaging.Driver'
+    classifier = 'externalLauncher'
+    from(project.convention.getPlugin(JavaPluginConvention).sourceSets.main.output)
+    configurations = [project.configurations.runtime]
+    exclude('META-INF/INDEX.LIST', 'META-INF/*.SF', 'META-INF/*.DSA', 'META-INF/*.RSA')
+
+    // Here you can add other Jar properties like destinationDir, for example
+    appendix = 'launcher'
+    archiveName = "${baseName}-${appendix}.${extension}"
+
+    // Dependencies included in resulting jar file
+    dependencies {
+        // This has to be specific list of dependencies
+        // Do not forget that Sparkling Water App is intended
+        // to run inside Spark environment providing some jars
+        // (like hadoop, aws) on classpath
+        include(dependency("ai.h2o:sparkling-water-core_${scalaBinaryVersion}"))
```

Comment:
Didn't see in the ShadowJar doc any methods that would allow inclusion of a sequence of dependencies -> haven't figured out how to extract that list of dependencies to a variable (so that I can append to it, the externalLauncher has some additional ones).